README.md

```diff
@@ -1,11 +1,94 @@
 # Nextflow pipeline for running the bioBakery
 
-by Kevin Bonham, PhD
+by Kevin Bonham, PhD 
 
-- `KneadData`
-- `MetaPhlAn`
-- `HUMAnN`
+[bioBakery](https://github.com/biobakery): software, documentation, and tutorials for microbial community profiling (created and mantained by the Huttenhower lab)
 
-## Setup
+- [`KneadData`](https://github.com/biobakery/kneaddata): a data quality-control pipeline that removes host genomic data within our metagenomic samples. Particularly, this pipeline uses a database containing a reference human genome so that all human DNA is removed from the samples. Link to more information here: (https://huttenhower.sph.harvard.edu/kneaddata/).
```

Comment:
```suggestion
- [`KneadData`](https://github.com/biobakery/kneaddata): 
  a data quality-control pipeline that trims low quality reads
  and removes host genomic data within our metagenomic samples.
  Particularly, this pipeline uses a database containing a reference human genome
  so that all human DNA is removed from the samples.
  Link to more information here: (https://huttenhower.sph.harvard.edu/kneaddata/).
```

Reply:
In addition to adding a clause about trimming low quality reads, I'm also adding "semantic line breaks" (https://sembr.org). Basically, markdown (and latex and typst etc) treat vertical whitespace (line breaks) as spaces in the rendered text. By breaking lines into semantic units (roughly clauses), it makes future changes easier to review, since diffs work on the basis of lines